src/Native/SymSgdNative/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+project (SymSgdNative)
+
+set(SOURCES
+    SymSgdNative.cpp
+)
+
+if(WIN32)
+    find_library(MKL_LIBRARY Microsoft.ML.MklImports HINTS ${CMAKE_SOURCE_DIR}/../../packages/mlnetmkldeps/0.0.0.1/runtimes/win-x64/native)
```

Comment:
Like we discussed offline, we shouldn't be hard-coding these paths and version numbers into our CMake scripts.